portfolio/src/main/webapp/script.js

```diff
@@ -125,33 +125,68 @@ const loadComments = async (maxComments) => {
   container.innerHTML = '';
 
   // process each comment, and add it to the DOM.
-  // currently, comments are simply paragraphs embedded
-  // in a div container
   comments.map((comment) => {
     const div = document.createElement('div');
     const paragraph = document.createElement('p');
     const title = document.createElement('h3');
     const timestamp = document.createElement('span');
+    const deleteButton = document.createElement('button');
 
     container.appendChild(div);
     div.appendChild(title);
     div.appendChild(paragraph);
     div.appendChild(timestamp);
+    div.appendChild(deleteButton);
 
     div.classList.add('comment');
     title.classList.add('comment-title');
     paragraph.classList.add('comment-body');
     timestamp.classList.add('comment-timestamp');
-    
+
     div.id = comment.id;
-    title.innerText = comment.title;
+    title.innerText = comment.title || 'This comment does not have a title';
```

Comment:
Optional: Or should that be disallowed?

Reply:
adding a comment title is required. the inclusion of the default here was a short-term feature to allow comments posted prior to the inclusion of the title feature to be rendered properly. it'll be removed once titles have been fully merged and i repopulate the database with comments that have titles